setup.py

```diff
@@ -859,17 +863,31 @@ def detect_modules(self):
             libs = ['crypt']
         else:
             libs = []
-        exts.append( Extension('_crypt', ['_cryptmodule.c'], libraries=libs) )
+
+        if 'vxworks' not in host_platform:
+            exts.append( Extension('_crypt', ['_cryptmodule.c'], libraries=libs) )
+        elif self.compiler.find_library_file(lib_dirs, 'OPENSSL'):
+            libs = ['OPENSSL']
+            exts.append( Extension('_crypt', ['_cryptmodule.c'], libraries=libs) )
 
         # CSV files
         exts.append( Extension('_csv', ['_csv.c']) )
 
         # POSIX subprocess module helper.
-        exts.append( Extension('_posixsubprocess', ['_posixsubprocess.c']) )
+        if 'vxworks' in host_platform :
+            exts.append( Extension('_vxwapi', ['_vxwapi.c']) )
```

Comment:
This change must be reverted and only done in a PR adding _vxwapi.c.

Reply:
Reverted done.